spirv_cross.hpp

```diff
@@ -346,6 +351,10 @@ class Compiler
 	SPIRConstant &get_constant(uint32_t id);
 	const SPIRConstant &get_constant(uint32_t id) const;
 
+	// Recursively marks any constants referenced by the specified constant instruction as being used
```

Comment:
Does this need to be in the public interface?

Reply:
Moved to protected section.